app/src/main/java/org/connectbot/util/VolumePreference.java

```diff
@@ -46,20 +48,17 @@ public VolumePreference(Context context, AttributeSet attrs, int defStyle) {
 	}
 
 	private void setupLayout(Context context, AttributeSet attrs) {
+		setDialogLayoutResource(R.layout.volume_preference_dialog_layout);
 		setPersistent(true);
 	}
 
 	@Override
-	protected View onCreateDialogView() {
-		SeekBar sb = new SeekBar(getContext());
-
-		sb.setMax(100);
-		sb.setProgress((int) (getPersistedFloat(
+	protected void onBindDialogView(View view) {
+		SeekBar volumeBar = (SeekBar) view.findViewById(R.id.volume_bar);
+		volumeBar.setProgress((int) (getPersistedFloat(
 				PreferenceConstants.DEFAULT_BELL_VOLUME) * 100));
-		sb.setPadding(10, 10, 10, 10);
-		sb.setOnSeekBarChangeListener(this);
-
-		return sb;
+		volumeBar.setOnSeekBarChangeListener(this);
+		super.onBindDialogView(view);
```

Comment:
I had seen a lot of examples calling the super class's method at the end but tested both ways and does not seem to have made any difference, for convention I moved to the first line of the method.